tests/screenshot/apps/components/ProgressBar.js

```diff
@@ -1,8 +1,9 @@
 import ProgressBar, {ProgressBarTooltip as Tooltip} from '../../../../ProgressBar';
 
-// TODO: RTL
+import {withConfig} from './utils';
+
 // [GT-28227] - generic to test Progress Bar and Background Progress Bar Colors - no end marked for this test
-const ProgressBarTests = [
+const ProgressBarLTRTests = [
 	<ProgressBar />,
 	<ProgressBar highlighted />,
```

Comment:
Can you add a test for disabled as well?

Reply:
👍